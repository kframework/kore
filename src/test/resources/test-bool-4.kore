[]
module BOOL
    hooked-sort Bool{} [hook{}("BOOL.Bool")]
    alias falseBool{}() : Bool{} where falseBool{}() := \dv{Bool{}}("untrue") []
endmodule
[]
