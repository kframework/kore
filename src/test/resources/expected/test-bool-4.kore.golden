Parse error: Error in module 'BOOL' -> alias 'falseBool' declaration (../src/test/resources//test-bool-4.kore 4:11) -> \dv (../src/test/resources//test-bool-4.kore 4:61) -> Verifying builtin sort 'BOOL.Bool' -> While parsing domain value: <string literal>:1:1:
  |
1 | untrue
  | ^^^^^
unexpected "untru"
expecting "false" or "true"
